### DISCUSSION TITLE
Changing average_scheduled_poll_interval

### DISCUSSION THREAD

**Discussion by kapso** [sidekiq/sidekiq#5427] [General]
Is there any downside to changing this value to `2` seconds? my understanding is that 5 seconds is the default.

We use the latest Pro version of Sidekiq - Sidekiq v6.5.1 / Sidekiq Pro v5.5.1

```ruby
Sidekiq.configure_server do |config|
  # Default is 5
  config.average_scheduled_poll_interval = 2
end
```

**Comment by mperham** [comment]
The downside is just potentially more load on Redis. Depending on the size
of your install it may or may not be noticeable.
